### PR TITLE
Move to unversioned python and add documentation

### PR DESCRIPTION
**Pull request by ComputersWithTimo**
I made these changes over the last few years while running a few minecraft servers for me and my friends.
Apparently i haven't made a PR yet.
The commits are somewhat crappy since I was debugging issues inside github and then pulling it onto the servers ^^

### CODE REVIEW

File: stats_mcserver.py [hunk 1]
```diff
@@ -1,22 +1,35 @@
...
 import sys
 
 ###sys.argv[1] - check type
 ###sys.argv[2] - ip address
 ###sys.argv[3] - port
 
+
 # Get server instance
-server = MinecraftServer(sys.argv[2], int(sys.argv[3]))
+try:
```
Comment (ComputersWithTimo) [inline stats_mcserver.py:11]: I also made changes to implement the new mcstatus "JavaServer" class as well as the old one.

File: zbx_minecraft_template.xml [hunk 2]
```diff
@@ -34,6 +34,13 @@
                             <name>Minecraft</name>
                         </application>
                     </applications>
+                    <triggers>
```
Comment (ComputersWithTimo) [inline zbx_minecraft_template.xml:37]: It also has a trigger that can notify you if a server goes offline.

**Comment by iTimo01** [comment]
This should also fix #4

File: stats_mcserver.py (hunk 1 above)
Comment (ComputersWithTimo) [reply stats_mcserver.py:11]: This should also fix #4

**Pull request merged**